src/pgduckdb_metadata_cache.cpp

```diff
@@ -75,7 +75,7 @@ uint32 schema_hash_value;
  * IsExtensionRegistered for details).
  */
 static void
-InvalidateCaches(Datum arg, int cache_id, uint32 hash_value) {
+InvalidateCaches(Datum, int, uint32 hash_value) {
```

Comment:
I don't think I like this type of change. If we explicitly don't use arguments provided by a hook, I'd like to know what those arguments are used for. Both for reviewing code, but also for when wanting to change behavior of a hook in the future.